tests/keras_tests/function_tests/test_model_gradients.py

```diff
@@ -89,29 +92,40 @@ def model_with_output_replacements(input_shape):
     return keras.Model(inputs=inputs, outputs=outputs)
 
 
-def representative_dataset():
-    yield [np.random.randn(1, 8, 8, 3).astype(np.float32)]
+def representative_dataset(num_of_inputs=1):
+    yield [np.random.randn(1, 8, 8, 3).astype(np.float32)]*num_of_inputs
+
+def _get_normalized_hessian_trace_approx(graph, interest_points, keras_impl, alpha, num_of_inputs=1):
```

Comment:
If this part is extracted from the model_gradients then it shouldn't be part of this test.
Changed the test's assertion to fit the expected results without normalization